addons/website_event_sale/static/src/scss/website_event_sale_templates.scss

```diff
@@ -11,3 +11,18 @@
         text-decoration: underline;
     }
 }
+
+// Match the record cover style of website_event
+.o_wevent_sale_event {
```

Comment:
same remark here, if possible put into a namespace

Reply:
I thought `o_wevent_sale_event` was the most relevant parent for this one :thinking: